model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/AddressResponse.scala

```diff
@@ -1,322 +1,196 @@
 package uk.gov.ons.addressIndex.model.server.response
 
-import play.api.http.Status
+import java.util
+import com.sksamuel.elastic4s.{HitAs, RichSearchHit}
 import play.api.libs.json.{Format, Json}
-import uk.gov.ons.addressIndex.model.db.index.{NationalAddressGazetteerAddress, PostcodeAddressFileAddress}
+import uk.gov.ons.addressIndex.model.db.index.{HybridIndex, NationalAddressGazetteer, PostcodeAddressFile}
 import uk.gov.ons.addressIndex.crfscala.CrfScala.CrfTokenResult
 import scala.util.Try
+import Model.HybridResponse
 
-/**
-  * Contains the reply for address by uprn request
-  *
-  * @param response found content
-  * @param status   response status / message
-  * @param errors   encountered errors (or an empty list if there is no errors)
-  */
-case class AddressByUprnResponseContainer(
-  response: AddressByUprnResponse,
-  status: AddressResponseStatus,
-  errors: Seq[AddressResponseError] = Seq.empty[AddressResponseError]
-)
+object Model {
 
-object AddressByUprnResponseContainer {
-  implicit val addressByUprnResponseContainerFormat: Format[AddressByUprnResponseContainer] =
-    Json.format[AddressByUprnResponseContainer]
-}
+  case class HybridResponse(
+    uprn: String,
+    lpi: Option[Seq[NationalAddressGazetteer]],
+    paf: Option[Seq[PostcodeAddressFile]],
```

Comment:
The format param dictates the columns selected from elastic (we dont want to throw loads of data over the network)
This is why the fields are Optional